modules/reference/pages/rpk/rpk-cluster/rpk-cluster-quotas.adoc

```diff
@@ -0,0 +1,37 @@
+= rpk cluster quotas
+:description: rpk cluster quotas
+
+Manage Redpanda client quotas.
+
+== Usage
+
+[,bash]
+----
+rpk cluster quotas [command] [flags]
+----
+
+== Aliases
+
+[,bash]
+----
+quotas, quota
+----
+
+== Flags
+
+[cols="1m,1a,2a"]
+|===
+|*Value* |*Type* |*Description*
+
+|--format |string |Output format. Possible values: `json`, `yaml`, `text`, `wide`, `help`. Default: `text`.
+
+|-h, --help |- |Help for quotas.
+
+|--config |string |Redpanda or `rpk` config file; default search paths are `/var/lib/redpanda/.config/rpk/rpk.yaml`, `$PWD/redpanda.yaml`, and `/etc/redpanda/redpanda.yaml`.
```

Comment:
```suggestion
|--config |string |Redpanda or `rpk` configuration file. Default search paths are `/var/lib/redpanda/.config/rpk/rpk.yaml`, `$PWD/redpanda.yaml`, and `/etc/redpanda/redpanda.yaml`.
```

Reply:
Rejecting this change for consistency, as we would need to apply it globally. If necessary we can do it in another PR.